src/ImageProcessorCore/Formats/Jpg/JpegDecoderCore.cs

```diff
@@ -884,6 +887,37 @@ private void processDRI(int n)
             ri = ((int)tmp[0] << 8) + (int)tmp[1];
         }
 
+        private void processApp1Marker(IList<ImageProperty> properties, int n)
+        {
+            readFull(tmp, 0, 6);
+            n -= 6;
+
+            var exif = tmp[0] == 'E' && tmp[1] == 'x' && tmp[2] == 'i' && tmp[3] == 'f' && tmp[4] == '\0' && tmp[5] == '\0';
+
+            // n should be the number of bytes used by the tiff image. However, the input stream
+            // has already moved past the start of the tiff image... because it reads 4096 chunks at a time.
+
+            var currPos = inputStream.Seek(0, SeekOrigin.Current); // mark our position in the stream
+
+            var tiffPosition = currPos - (4096 - bytes.i);
+
+            inputStream.Seek(tiffPosition, SeekOrigin.Begin);
+
+            // we may not have a tiff stream... so double check that we do
+            TiffDecoderCore tiffDecoder = TiffDecoderCore.Create(inputStream);
+            if (tiffDecoder != null)
+            {
+                tiffDecoder.Decode();
+
+                // got em...
+                tiffDecoder.FillExifProperties( properties );
```

Comment:
This is where it currently is. The EXIF properties are extracted into the generic image properties list. You can also see examples in the Tests. Next steps are probably to clean it up, comment, harden it while adding more tests.